plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -39,33 +41,39 @@ public void downloadOldSchema() throws Exception {
 
     @Test
     public void validateSchema() throws IOException {
-        String schema = generateSchema();
-        JSONObject jsonSchema = new JSONObject(
-            new JSONTokener(schema));
-
-        String answer = "";
-        try {
-            answer = Util.toStringFromYamlFile(this, "merge3.yml");
-        } catch (URISyntaxException e) {
-            e.printStackTrace();
-        } catch (IOException e) {
-            e.printStackTrace();
-        }
-
-        System.out.println(answer);
+        JSONObject schemaObject  = generateSchema();
```

Comment:
@casz After going through the trouble of converting the building of the Schema using JSON Objects.I ran it and bang! Stack OverFlow Error !

Reply:
`java.lang.StackOverflowError
	at org.json.JSONObject.length(JSONObject.java:806)`.
Looks like a silly mistake somewhere I cannot spot with my bare eyes :cry: